### PR TITLE
Suggest what to use for deprecated attribute.

### PR DESCRIPTION
**Pull request by cmarqu**

### CODE REVIEW

File: cocotb/handle.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def __setattr__(self, name, value):
     def __getattr__(self, name):
         if name in self._compat_mapping:
             if name not in _deprecation_warned:
-                warnings.warn("Use of %s attribute is deprecated" % name)
+                warnings.warn("Use of attribute '%s' is deprecated, use '%s' instead" % (name, self._compat_mapping[name]))
                 _deprecation_warned[name] = True
```
Comment (eric-wieser) [inline cocotb/handle.py]: Better to use `%r` instead of `'%s'`, it adds quotes for you

Comment (cmarqu) [reply cocotb/handle.py]: Done, thanks.

File: cocotb/handle.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def __str__(self):
     def __setattr__(self, name, value):
         if name in self._compat_mapping:
             if name not in _deprecation_warned:
-                warnings.warn("Use of %s attribute is deprecated" % name)
+                warnings.warn("Use of attribute %r is deprecated, use %r instead" % (name, self._compat_mapping[name]))
                 _deprecation_warned[name] = True
```
Comment (eric-wieser) [inline cocotb/handle.py]: Out of scope for this patch, but this should really be
```
warnings.warn("...", DeprecationWarning, stacklevel=2)
```
(leaving python to deal with the "already-warned" state)

We'll want to fix #1194 before we make that change.

**Comment by cmarqu** [comment]
We may even want to remove the "compatibility mapping" altogether, those attributes have been deprecated for quite a while now.

**Pull request merged**